datahub-web-react/src/app/settings/posts/constants.ts

```diff
@@ -0,0 +1,6 @@
+export const TITLE_FIELD_NAME = 'title';
+export const DESCRIPTION_FIELD_NAME = 'description';
+export const LINK_FIELD_NAME = 'link';
+export const LOCATION_FIELD_NAME = 'location';
+export const TYPE_FIELD_NAME = 'switch';
```

Comment:
it's called "switch"? does type not work?

Reply:
it does lol it used to be called `SWITCH_FIELD_NAME` but i changed it and forgot to update the value here (even though technically it doesn't matter cuz it's just a const that's used) - but updating!